modules/private-shared-lib/src/events/outbound_domain_event/party_info_callback_processed.ts

```diff
@@ -18,50 +18,51 @@
  * Gates Foundation
  - Name Surname <name.surname@gatesfoundation.com>
  * Modusbox
- - Yevhen Kyriukha <yevhen.kyriukha@modusbox.com>
```

Comment:
?

Reply:
Not sure why it got deleted, may be during merging with upstream. Re-adding :)